pkg/updater/updater.go

```diff
@@ -58,7 +60,8 @@ func (g *OSS) Fetch() (io.Reader, error) {
 	data := &FormData{
 		OS:             runtime.GOOS,
 		Arch:           runtime.GOARCH,
-		CurrentVersion: version.BuildVersion,
+		CurrentVersion: g.CurrentVersion,
```

Comment:
curious: not familiar with this part of the code base, why do we need to change `currentVersion` ?

Reply:
It wasn't being used anywhere, so we figured it was probably meant to be used here. `g.CurrentVersion` is always set to `version.BuildVersion` anyway so it should behave the same.

The alternative is to delete `g.CurrentVersion` and just always use `version.BuildVersion`, which might be better.

Reply:
if we're not using it anywhere else and `version.BuildVersion` carries the same data i vote delete